plugins/gradle/src/main/java/org/wildfly/swarm/plugin/gradle/PackageTask.java

```diff
@@ -66,6 +71,23 @@ public void packageForSwarm() throws Exception {
 
         GradleArtifactResolvingHelper resolvingHelper = new GradleArtifactResolvingHelper(project);
         Properties propertiesFromExtension = getPropertiesFromExtension();
+        List<File> moduleDirs = getModuleDirs();
+        if (moduleDirs.isEmpty()) {
```

Comment:
https://github.com/SetoKaiba/wildfly-swarm/blob/ef7b403f6a930f9102b823eb80575b0ff3cbb471/plugins/gradle/src/main/java/org/wildfly/swarm/plugin/gradle/SwarmExtension.java#L114-L117

Maybe you forget to check the unmodified code. The setter of moduleDirs.